element_lab/lab.py

```diff
@@ -16,14 +16,18 @@ def activate(schema_name, create_schema=True, create_tables=True):
 @schema
 class Lab(dj.Lookup):
     definition = """
-    lab             : varchar(24)  #  Abbreviated lab name 
+    lab             : varchar(24)    #  Abbreviated lab name
     ---
     lab_name        : varchar(255)   # full lab name
     institution     : varchar(255)
     address         : varchar(255)
     time_zone       : varchar(64)
     """
 
+    def make_nwb(cls, lab_key):
+        from .export import lab_to_nwb
```

Comment:
I may be mistaken but I don't see a `lab_to_nwb` function within `export/nwb.py`.

Reply:
Good catch! This was a holdover from a previous version. Current version only makes dicts to feed into a user's session-level nwb generation process.